rajawali/src/main/java/c/org/rajawali3d/engine/RenderView.java

```diff
@@ -0,0 +1,66 @@
+package c.org.rajawali3d.engine;
+
+import android.graphics.Rect;
+import android.support.annotation.Nullable;
+import c.org.rajawali3d.annotations.GLThread;
+
+/**
+ * Interface to be implemented by views that are to be managed by the Engine. A RenderView represents
+ * a viewport on the render surface
+ *
+ * @author Jared Woolston (Jared.Woolston@gmail.com)
+ */
+public interface RenderView {
```

Comment:
Yeah I don't think its going to be too much of a can of worms, but definitely enough work to save it for another version. Lets at least make sure the information is available in the meantime in case a developer wants to do something on their own.